docs/source/about/advanced_usage.rst

```diff
@@ -414,6 +414,41 @@ dwmflush
 
       dwmflush = enabled
 
+unpaced
+^^^^^^^^
+
+**Description**
+   Don't try to match client frame rate and stream every frame the host produces as soon as possible.
+   Can significantly improve frame time stability.
+
+   .. Caution:: Applies to Windows only. Experimental option.
+
+**Default**
+   ``disabled``
+
+**Example**
+   .. code-block:: text
+
+      unpaced = enabled
```

Comment:
All existing examples all show how to set it to the default value

```suggestion
      unpaced = disabled
```

Reply:
Fixed https://github.com/LizardByte/Sunshine/pull/1203/commits/a8a53d6dc925bab2ce1537550487c3de30fd79e2